pkg/handler/crud.go

```diff
@@ -253,6 +255,7 @@ func (c *crud) PutFlag(params flag.PutFlagParams) middleware.Responder {
 	resp.SetPayload(payload)
 
 	entity.SaveFlagSnapshot(getDB(), util.SafeUint(params.FlagID), getSubjectFromRequest(params.HTTPRequest))
+	notify.All(getDB(), util.SafeUint(params.FlagID), notify.UPDATED, notify.FLAG, getSubjectFromRequest(params.HTTPRequest))
```

Comment:
A note to myself for the idea of future refactoring - this is a typical hook pattern for both saving the flagSnapshot and notifyAll, maybe good to extract that in the future.